docusaurus/package.json

```diff
@@ -19,6 +19,7 @@
     "@mdx-js/react": "^1.6.22",
     "clsx": "^1.2.1",
     "prism-react-renderer": "^1.3.5",
+    "raw-loader": "^4.0.2",
```

Comment:
Commit message title nit: Maybe "*the* `raw-loader`" (so the proposal is to also use backticks for all `raw-loader` occurrences).